spec/features/create_object_h2_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Use H2 to create an object', type: :feature do
+  let(:collection_title) { RandomWord.nouns.next }
+  let(:item_title) { "SUL Logo for #{collection_title}" }
+  let(:start_url) { "#{Settings.h2_url}/dashboard" }
+  let(:user_email) { "#{AuthenticationHelpers.class_variable_get(:@@username)}@stanford.edu" }
+
+  before do
+    authenticate!(start_url: start_url,
+                  expected_text: 'Your collections')
+  end
+
+  scenario do
+    click_link '+ Create a new collection'
+
+    # Checks for specific content on create collection view
+    expect(page).to have_content('Manage release of deposits for discovery and download')
+
+    # Fills in basic collection information and saves
+    fill_in 'Collection name', with: collection_title
+    fill_in 'Description', with: "Integration tests for #{collection_title}"
+    fill_in 'Contact email', with: user_email
+
+    # Adds user to depositor field
+    fill_in 'Depositors', with: AuthenticationHelpers.class_variable_get(:@@username)
+
+    click_button 'Deposit'
+
+    expect(page).to have_content(collection_title)
+
+    # Deposit an item to the collection
+    find('header', text: collection_title).sibling('button').click
```

Comment:
Can you do `click_button '+ Deposit to this collection', match: :first` instead?

Reply:
Do we care that adding a deposit is in the same collection we created? On stage (for my login) there is a long list of collections (see below screenshot) and this was how I could ensure that the object being deposited was using the same collection that was just created.

![Screen Shot 2020-11-03 at 4 13 55 PM](https://user-images.githubusercontent.com/71847/98050750-9a6a9e80-1def-11eb-9b83-90e2ee3cca0b.png)

Reply:
Ah, that's a good point.